README.md

```diff
@@ -2,4 +2,370 @@
+
+        python $CONDA_PREFIX/scripts/isis3VarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data directory]
+                
+
+    Directions for running rsync to download ISIS3 data can be found [here.](#ISIS3DataDownload) Now everytime the isis3 environment is activated, $ISISROOT, $ISIS3DATA, and $ISIS3TESTDATA will be set to the values passed to isis3VarInit.py. This does not happen retroactively, re-activate the isis3 envionment with the following command:
+
+        #Depending on your OS, you may need to use conda activate isis3 instead
+        source activate isis3
+                
+
+### Operating System Requirements
+
+ISIS3 runs on many UNIX variants. ISIS does not run natively on MS Windows, although it has been successfully run on Windows 10 using the Windows Subsystem for Linux (WSL). Instructions for doing this can be found [here.](#RunningOnWindows) The UNIX variants ISIS3 has been successfully built on are:
+
+-   Ubuntu 18.04 LTS
+-   Mac OS X 10.13.6 High Sierra
+-   Fedora 28
+-   CentOS 7.2
+
+ISIS3 may be run on other Linux or macOS operating systems then those listed above, but it has not been tested and is not supported.
+
+### Hardware Requirements
+
+Here are the minimum hardware requirements
+
+-   64-bit (x86) processors
+-   2 GB RAM
+-   2.5 GB of disk space for ISIS3 binaries
+-   10 GB to 510 GB disk space for ISIS3 data
+-   10 GB to many TB disk space for processing images
+-   A quality graphics card
+
+To build and compile ISIS3 requires following the instructions listed below, which are given on the GitHub wiki page for the ISIS3 project:
+
+-   [Getting Started With GitHub](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#getting-started-with-github)
+-   [Building ISIS3 With cmake](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-isis3)
+-   [New ISIS3 environmental variables and their meanings](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#new-environmental-variable-meanings)
+-   [Custom data and test directories](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#custom-data-and-test-data-directories)
+-   [Cleaning builds](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#cleaning-builds)
+-   [Building individual applications/objects](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-individual-isis3-applicationsobjects)
+-   [Building ISIS3 documentation](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-isis3-documentation)
+-   [What to do if you encounter any problems](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#problems)
+
+<span id="RunningOnWindows"></span>
+
+### Running ISIS3 on Windows 10
```

Comment:
I feel like, for now, getting ISIS 3.6.0 is a no go for Windows (WSL). I can't get past and opecv install error no matter what I try. Perhaps simply remove until we can figure it out. Or mention the rsync method for ISIS v3.5.2 works for WSL (see blog) and we will update this once we figure it out. 

```
$ cubeatt
cubeatt: error while loading shared libraries: libopencv_core.so.3.2: cannot enable executable stack as shared object requires: Invalid argument
```

Reply:
:+1: